hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/file/tfile/TestTFileSeqFileComparison.java

```diff
@@ -516,7 +516,7 @@ private static String parameters2String(MyOptions options) {
   }
 
   private static class MyOptions {
-    String rootDir = GenericTestUtils.getTestDir().getAbsolutePath();;
+    private String rootDir = GenericTestUtils.getTestDir().getAbsolutePath();;
```

Comment:
no need to add `private` here as part of this change

Reply:
I will fix it.